Mathlib/RingTheory/Algebraic.lean

```diff
@@ -193,6 +193,29 @@ theorem Transcendental.pow {r : A} (ht : Transcendental R r) {n : ℕ} (hn : 0 <
     Transcendental R (r ^ n) := fun ht' ↦ ht <| ht'.of_pow hn
 #align transcendental.pow Transcendental.pow
 
+lemma IsAlgebraic.invOf {x : S} [Invertible x] (h : IsAlgebraic R x) : IsAlgebraic R (⅟ x) := by
+  obtain ⟨p, hp, hp'⟩ := h
+  refine ⟨p.reverse, by simpa using hp, ?_⟩
+  rwa [Polynomial.aeval_def, Polynomial.eval₂_reverse_eq_zero_iff, ← Polynomial.aeval_def]
+
+lemma IsAlgebraic.invOf_iff {x : S} [Invertible x] :
```

Comment:
I would suggest this (and similarly for `IsAlgebraic.inv_iff`):
```suggestion
lemma IsAlgebraic.invOf_iff (x : S) [Invertible x] :
```

Reply:
I think these iffs usually have implicit arguments?